fern/versions/public.yml

```
@@ -113,6 +113,33 @@ navigation:
           - page: Snap-in triggered by an external source
             slug: triggered-external-source
             path: ../docs/pages/tutorials/triggered-external-source.mdx
+      - section: ADaaS
+        slug: adaas
+        contents:
+          - page: "Airdrop-as-a-Service (ADaaS)"
+            slug: overview
+            path: ../docs/pages/adaas/overview.mdx
+          - page: "Getting Started"
+            slug: getting-started
+            path: ../docs/pages/adaas/getting-started.mdx
+          - page: "Extraction Phases"
+            slug: extraction-phases
+            path: ../docs/pages/adaas/extraction-phases.mdx
+          - page: "Extracting External Sync Units"
```

Start the titles with the object, not the action. Also suggest changing the filenames to match.
```suggestion
          - page: "External sync unit extraction"
```

---

I have fixed all titles in the public.yml, as well as modified the filenames and slugs.